deliverables/README.md

```diff
@@ -2,7 +2,10 @@
 ###Intro to the Back End
 ####Required
 What is the difference between front- and back-end development?<br>
-Why do you need to install Ruby but not HTML/CSS?
+Front end development is the graphcis that people see such as font size or colors, while back end development is whats going on behind the scenes or how you interact with the front end. 
+
+Why do you need to install Ruby but not HTML/CSS
+Since Ruby has a lot of unique commands, it is essential to install it or the commands will not be understood. 
```

Comment:
Because you already have an "interpreter" installed for HTML, CSS, and Javascript:  your browser knows how to read these languages and translate them into behavior.